### ISSUE TITLE
Nonce not added to Meta tag

### ISSUE THREAD

**Issue by twustep** [cloudgrayau/csp#2]
Hello, thanks for creating such a great plugin. 

I have installed the CSP plugin to work along side SEOMatic. 

It all works as intended. I have google tags and facebook tags rendering through SEOMatic, the inline tags correctly render with a nonce. CSP Plugin then reads those nonces and sets them in the Meta Tags. This all works as expected. 

However, I have multiple sites in the craft CMS that all use the same layout.twig and CMS settings. Each site gets its own google and facebook tag. I can see that SEOMatic correctly renders the nonces for the tags. But the nonces are NOT present on the meta tags which causes a CSP violation.

In other words, it seems like SEOMatic and CSP Plugin work correctly on the main site but not on the secondary sites. 

From what I can see SEOMatic is configured correctly and I suspect I either don't have something set on the CSP plugin side or perhaps code needs to be modified to support this. 

I am happy to provide urls and/or more information. I appreciate your help in advance! Thank you!

**Comment by cloudgrayau** [comment]
I will have a look. I assume by the "Google" tags and "Facebook" tags you mention, you are referring to the tracking codes in SEOMatic?

**Comment by twustep** [comment]
Yes, that is correct. The Google and Facebook tags are the ones I setup in SEOMatic and render in the markup. Each site has their own Tag IDs (facebook and gtm) that are configured in the CMS. 

Thank you so much!

**Comment by cloudgrayau** [comment]
I setup a fresh install with two sites as a multi-site setup, each with their own Google and Facebook tags. After testing, neither site had issues rendering the correct nonce tags.

Do you have a test website I can see? Do you happen to have any caching involved that could be potentially effecting the results?

Thanks,

**Comment by twustep** [comment]
Thank you so much for looking into this. 

It turns out as you correctly pointed out it had to do with caching. The site in question has various levels of caching (at the server level and then app level). Clearing all caches and letting them re-hydrate did the trick. Now all five sites work as expected. 

Apologies for the false alarm. I really appreciate you looking into this.
